### DISCUSSION TITLE
Zoe Q210 R-Link 1 - full screen rear camera

### DISCUSSION THREAD

**Discussion by HK-Zoe** [cedricp/ddt4all#835] [Q&A]
Hi all!
Is there any way to make full screen rear camera on R-Link 1 (Zoe Q210) when im on "R" with ddt4all?

**Comment by benoitsauvard** [comment]
Hi, have you find how to turn on fullscreen? I want to to enable it also.

**Comment by AndyIreland** [comment]
only this similar - but it most probably what you are looking for:
https://www.navinc.nl/9-tv-in-motion/video-in-motion-module-renault-r-link-system-can
